exporter/exporter.go

```diff
@@ -90,11 +128,44 @@ func NewNebulaExporter(ns string, listenAddr string, client *kubernetes.Clientse
 	}
 
 	graphdLabels := []string{
-		"create_edge", "fetch_vertices", "parse",
-		"create_tag", "all", "create_space",
-		"drop_space", "go", "insert_edge",
-		"storageClient", "use", "insert_vertex",
-		"show", "yield",
+		"storageClient",
+		"metaClient",
+		"all",
+		"use",
+		"show",
+		"find_path",
+		"create_edge",
+		"create_tag",
+		"create_snapshot",
+		"delete_edge",
+		"delete_vertices",
```

Comment:
delete vertex

Reply:
get graph_delete_vertices_*  from graphd, I check it again